JLS-API-Client.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
 
-  s.name         = "JLS-API-Client"
+  s.name         = "JLSAPIClient"
```

Comment:
Maybe, having so many uppercase letter without any separation is not the best idea. Should we name it `JlsApiClient`? Or `JLSApiClient` would be fine with me, too ... that's why I named it this way in the first place.

Reply:
JLSApiClient would be fine, JLS would then read like a prefix namespace. I have removed the hyphens because Xcode cannot handle them correctly.